.github/workflows/ci-server.yaml

```diff
@@ -31,5 +31,11 @@ jobs:
         run: yarn nx lint twenty-server
       - name: Server / Run jest tests
         run: yarn nx test twenty-server
-      # - name: Server / Run e2e tests
-      #   run: yarn nx test:e2e twenty-server
+      - name: Server / Build
+        run: yarn nx build twenty-server
+      - name: Server / Write .env
+        run: |
+          cd packages/twenty-server
+          cp .env.example .env
```

Comment:
would be nice to have .env.test in the future eventually when we work on server-side tests